LaTeX/TeX.sublime-syntax

```diff
@@ -175,34 +175,54 @@ contexts:
     - include: paragraph-pop
 
   tex-constant-value:
-    - match: ((-?)\d+(\.)\d*)({{dimunits}}?)
-      scope: meta.number.float.decimal.tex
-      captures:
-        1: constant.numeric.value.tex
-        2: keyword.operator.arithmetic.tex
-        3: punctuation.separator.decimal.tex
-        4: constant.numeric.suffix.tex
-    - match: ((-?)\d+)({{dimunits}}?)
-      scope: meta.number.integer.decimal.tex
-      captures:
-        1: constant.numeric.value.tex
-        2: keyword.operator.arithmetic.tex
-        3: constant.numeric.suffix.tex
+    - match: (-?)\d+(\.)\d*
+      scope: constant.numeric.value.tex
+      captures:
+        1: keyword.operator.arithmetic.tex
+        2: punctuation.separator.decimal.tex
+      push:
+        - decimal-float-meta
+        - tex-dimension-suffix
+    - match: ((-?)\d+)
+      scope: meta.number.integer.decimal.tex constant.numeric.value.tex
```

Comment:
`meta.number` is duplicate here and the whole number seems scoped `keyword.operator`.

![grafik](https://user-images.githubusercontent.com/16542113/202898597-b537f701-de83-403c-ba2e-4ba1b8aeba67.png)

Reply:
fixed.  Btw, is there a good way to check for something like this in the automatic tests?